Formula/android-sdk.rb

```diff
@@ -55,6 +55,14 @@ def install
       EOS
     end
 
+    %w[sdkmanager].each do |tool|
```

Comment:
there's only one, so you can kill the outer loop and do `(bin/"sdkmanager").write`